.gitignore

```diff
@@ -7,3 +7,4 @@ venv/
 .venv/
 .pytest_cache/
 **__pycache__/
+db.sqlite3
```

Comment:
There is an extra space at the end of 'db.sqlite3'. This might cause the file not to be ignored as expected. Consider removing the space to ensure it is ignored correctly.